docs/catalog-information/v1_0/raster_profile.md

```diff
@@ -1,7 +1,7 @@
 # Raster Catalog Profile Information
 
 1. **typename** = `mc:MCRasterRecord`
-2. **main_namespace** = `http://schema.mapcolonies.com/raster`
+2. **main_namespace** = `http://schema.mapcolonies.com/raster` (This site doesn't exist)
```

Comment:
comment is unnecessary.
this is almost always the case for xml schema

Reply:
i think we should at least mention it - to prevent them searching for it